pandas/tests/frame/test_repr_info.py

```diff
@@ -27,6 +28,16 @@
 
 
 class TestDataFrameReprInfoEtc:
+    def test_repr_bytes_61_lines(self):
+        # GH#12857
+        lets = "ACDEFGHIJKLMNOP"
+        slen = 50
+        nseqs = 1000
+        words = [[random.choice(lets) for x in range(slen)] for _ in range(nseqs)]
+        df = pd.DataFrame(words).astype("S1")
+        assert (df.dtypes == "S1").all()
```

Comment:
hmm, do we actually allow this? meaning I *think* we *always* convert to object

Reply:
yah i thought this was a bit sketchy too.  ill pull this out so we can stick a pin in it while i mow down more of these "needs test" issues